R/footer_dfe_auto.R

```diff
@@ -0,0 +1,100 @@
+#' Footer Function
+#'
+#' This function create a gov style footer for your page
+#' @param full Whenever you want to have blank footer or official gov version.
+#' Defaults to \code{FALSE}
+#' @return a footer html shiny object
+#' @keywords footer
+#' @export
+#' @examples
+#' if (interactive()) {
+#'   ui <- fluidPage(
+#'     shinyGovstyle::header(
+#'       main_text = "Example",
+#'       secondary_text = "User Examples",
+#'       logo = "shinyGovstyle/images/moj_logo.png"
+#'     ),
+#'     shinyGovstyle::banner(
+#'       inputId = "banner", type = "beta", "This is a new service"
+#'     ),
+#'     tags$br(),
+#'     tags$br(),
+#'     footer_dfe(full = TRUE)
+#'   )
+#'
+#'   server <- function(input, output, session) {
+#'     observeEvent(input$link_to_accessibility, {
+#'       updateTabsetPanel(session, "navlistPanel", selected = "Accessibility")
+#'     })
+#'   }
+#'
+#'   shinyApp(ui = ui, server = server)
+#' }
+footer_dfe <- function(full = FALSE) {
```

Comment:
It'd be helpful to be able to pass here a dynamic list of link names that the user wants putting in. We'd probably want the default to be along the lines of `c("Accessibility", "Support and feedback", "Cookies")`

Here's an example function from an existing dashboard:
https://github.com/dfe-analytical-services/local-authority-interactive-tool/blob/bb117d6d063a2a37a01722c4955169cc534b5b54/R/fn_ui_layout.R#L215

And how it then looks in the UI:
https://github.com/dfe-analytical-services/local-authority-interactive-tool/blob/bb117d6d063a2a37a01722c4955169cc534b5b54/ui.R#L145